airbyte-integrations/connectors/source-close-com/acceptance-test-config.yml

```diff
@@ -4,7 +4,7 @@ connector_image: airbyte/source-close-com:dev
 acceptance_tests:
   spec:
     tests:
-      - spec_path: "source_close_com/spec.json"
+      - spec_path: "source_close_com/spec.yaml"
```

Comment:
I don't quite know why we're renaming these. Can you point me to where this ask is coming from?

Reply:
Not a requirement, just changed JSON to YAML for quickly pasting to manifest in the future :smile: